dlme_airflow/utils/add_iiif_v3_source.py

```diff
@@ -0,0 +1,54 @@
+# /bin/python
+import json
+import os
+import pandas
+import urllib.request
+
+
+def merge_dataframes(**kwargs):
+    """Called by the Airflow workflow to merge multiple data sources for one provider
+    e.g. when the data provider has IIIF and OAI-PMH and one source does not have all
+    of the metadata."""
+    collection = kwargs["collection"]
+    data_file = collection.datafile("json")
+    if os.path.isfile(data_file):
+        df1 = read_datafile_with_lists(data_file)
+        df2 = build_key_value_df(collection.catalog.metadata.get("iiif_collection_url"))
+        df = pandas.merge(df1, df2, on=["id"])
```

Comment:
This is basically the only think that isn't tested since we have tests for `read_datafile_with_lists` and `build_key_value_df`. Doesn't seem worth testing to me but let me know what you think.